ECIPs/ECIP-1000.md

```diff
@@ -0,0 +1,343 @@
+	ECIP: 1000
+	Title: ECIP process
+	Author: Wei Tang <hi@that.world>
+	Comments-Summary: No comments yet.
+	Comments-URI: https://github.com/ethereumclassic/ECIPs/issues/58
+	Status: Draft
+	Type: Process
+	Created: 2017-06-29
+	License: Apache-2
+
+# Abstract
+
+An Ethereum Classic Improvement Proposal (ECIP) is a design document providing information to the Ethereum Classic community, or describing a new feature for Ethereum Classic or its processes or environment. The ECIP should provide a concise technical specification of the feature and a rationale for the feature.
+
+We intend ECIPs to be the primary mechanism for proposing new features, for collecting community input on an issue, and for documenting the design decisions that have gone into Ethereum Classic. The **ECIP author** is responsible for building consensus within the community and documenting dissenting opinions.
+
+Because the ECIPs are maintained as text files in a versioned repository, their revision history is the historical record of the feature proposal.
+
+# Copyright
+
+This ECIP is licensed Apache-2, originally by [Luke Dashjr](https://github.com/luke-jr) under BSD 2-clause license.
```

Comment:
Is this right? Does luke-jr have a copyright over all improvement processes?

Reply:
if so, we need to link to the real license

Reply:
To confirm, is this line part of the ECIP by @TokenHash?

As far as I can tell, he just switched file formats to Markdown and then some basic boldening and title fixes. We can create a new issue for this specific point.

Reply:
yes, this is the original ECIP-1000 which I adjusted for formatting and alinement with:

• https://github.com/ethereumclassic/ECIPs/blob/master/README.md
• https://github.com/ethereumclassic/ECIPs/blob/master/ECIP-1.sample.md
• https://github.com/ethereumclassic/ECIPs/blob/master/ECIP-number.template.md
• https://github.com/ethereumclassic/ECIPs/blob/master/process.png

Also adjusted the status key:

• Draft
• Last call
• Accepted
• Final
• Deferred
• Rejected
• Withdrawn
• Replaced

But, I did not change content, basic structure or licensing.